Gemfile.lock

```diff
@@ -328,9 +321,7 @@ GEM
     sterile (1.0.14)
       nokogiri
     stringex (2.8.5)
-    stripe (4.24.0)
-      faraday (~> 0.13)
-      net-http-persistent (~> 3.0)
+    stripe (5.1.0)
     thor (0.20.3)
     thread_safe (0.3.6)
     tilt (2.0.9)
```

Comment:
Can you delete this whole file's diff from your PR, please? 
To keep your PR focused on just your changes, and to leave the gem version bumps to Dependabot.